### ISSUE TITLE
Cannot build flutter stripe in my project

### ISSUE THREAD

**Issue by FullStack1106** [flutter-stripe/flutter_stripe#1103]
I followed everything to install this library, but I've got this error when build & run project.

e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (53, 26): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (54, 34): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (55, 35): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (56, 23): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (58, 72): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (64, 65): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (57, 26): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (58, 34): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (59, 35): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (60, 23): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (62, 52): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (73, 45): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (84, 18): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (85, 26): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (86, 27): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (87, 15): Unresolved reference: getValOr

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe_android:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 48s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by FullStack1106** [comment]
minSdkVersion 21
ext.kotlin_version = '1.6.10'

import io.flutter.embedding.android.FlutterFragmentActivity
class MainActivity: FlutterFragmentActivity() {
}


 <style name="NormalTheme" parent="Theme.MaterialComponents">
        <item name="android:windowBackground">?android:colorBackground</item>
    </style>

dependencies {
        classpath 'com.android.tools.build:gradle:7.2.2'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }

Followed everything, but what is issue?

**Comment by remonh87** [comment]
did you run `flutter clean`  before?

**Comment by FullStack1106** [comment]
yes, did

**Comment by remonh87** [comment]
I added 8.0 to my project and have the same config as you and it did build without problems. Maybe ./gradlew clean will help .

I use grade-wrapper: `https\://services.gradle.org/distributions/gradle-7.4-all.zip`

**Comment by FullStack1106** [comment]
I use same gradle wrapper too.
I will try to clean gradlew and try again.

**Comment by FullStack1106** [comment]
I got this error message when run ./gradlew clean in android folder.

penJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x0000000604400000, 532676608, 0) failed; error='The paging file is too small for this operation to complete' (DOS error/errno=1455)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 532676608 bytes for Failed to commit area from 0x0000000604400000 to 0x0000000624000000 of length 532676608.
# An error report file with more information is saved as:
# F:\BaTransfer\ba_transfer_customer\android\hs_err_pid32280.log


but when I run this command in another kotlin project, it is works.
do you know what is issue?

**Comment by FullStack1106** [comment]
I did "./gradlew clean" after restart pc.
but same error.

e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (53, 26): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (54, 34): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (55, 35): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (56, 23): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (58, 72): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardFormPlatformView.kt: (64, 65): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (57, 26): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (58, 34): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (59, 35): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (60, 23): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (62, 52): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\flutter\stripe\StripeSdkCardPlatformView.kt: (73, 45): Unresolved reference: it
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (84, 18): Unresolved reference: getValOr
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (85, 26): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (86, 27): Unresolved reference: getIntOrNull
e: F:\Android\flutter\.pub-cache\hosted\pub.dartlang.org\stripe_android-8.0.0\android\src\main\kotlin\com\reactnativestripesdk\CardFieldViewManager.kt: (87, 15): Unresolved reference: getValOr

**Comment by FullStack1106** [comment]
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe_android:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

**Comment by remonh87** [comment]
do you have added a stripe dependency in build.gradle?

**Comment by FullStack1106** [comment]
no , I did not.

**Comment by FullStack1106** [comment]
I did not see about that in document.
Please let me know how to add it.

**Comment by asaarnak** [comment]
Same error for me after updating to flutter_stripe 8.0.0+1.
```
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardFormPlatformView.kt:53:26 Unresolved reference: getValOr
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardFormPlatformView.kt:54:34 Unresolved reference: getIntOrNull
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardFormPlatformView.kt:55:35 Unresolved reference: getIntOrNull
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardFormPlatformView.kt:56:23 Unresolved reference: getValOr
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardFormPlatformView.kt:58:72 Unresolved reference: it
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardFormPlatformView.kt:64:65 Unresolved reference: it
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardPlatformView.kt:57:26 Unresolved reference: getValOr
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardPlatformView.kt:58:34 Unresolved reference: getIntOrNull
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardPlatformView.kt:59:35 Unresolved reference: getIntOrNull
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardPlatformView.kt:60:23 Unresolved reference: getValOr
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardPlatformView.kt:62:52 Unresolved reference: it
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/flutter/stripe/StripeSdkCardPlatformView.kt:73:45 Unresolved reference: it
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/reactnativestripesdk/CardFieldViewManager.kt:84:18 Unresolved reference: getValOr
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/reactnativestripesdk/CardFieldViewManager.kt:85:26 Unresolved reference: getIntOrNull
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/reactnativestripesdk/CardFieldViewManager.kt:86:27 Unresolved reference: getIntOrNull
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-8.0.0/android/src/main/kotlin/com/reactnativestripesdk/CardFieldViewManager.kt:87:15 Unresolved reference: getValOr

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe_android:compileReleaseKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details
```

**Comment by FullStack1106** [comment]
it is worked when you use previous version?
If yes, please let me know which version did you use before.
I will use it for now.

**Comment by asaarnak** [comment]
Yes, 7.0.0 worked.

**Comment by FullStack1106** [comment]
> do you have added a stripe dependency in build.gradle?

I added this line in dependencies of build.gradle.
implementation 'com.stripe:stripe-android:20.19.1'
Bug same error yet.

**Comment by asaarnak** [comment]
I just updated flutter_stripe version from 7.0.0 to 8.0.0+1, no other changes.

**Comment by remonh87** [comment]
can you try to build the example app of this repo and see if the issue persist? else I need to have a reproduction repo to properly debug it

**Comment by FullStack1106** [comment]
I've changed 8.0.0+1 to 7.0.0.
I built project.
but it crashed when create payment intent.

**Comment by FullStack1106** [comment]
PS F:\Stripe Flutter\example> flutter pub get
Warning: pubspec.yaml has overrides from pubspec_overrides.yaml
Because stripe_example depends on stripe_js from path which doesn't exist (could not find package stripe_js at "..\packages\stripe_js"), version solving failed.
Running "flutter pub get" in example...                                 
pub get failed (66; Because stripe_example depends on stripe_js from path which doesn't exist (could not find package stripe_js at "..\packages\stripe_js"), version solving failed.)
PS F:\Stripe Flutter\example>

**Comment by asaarnak** [comment]
@remonh87 
Ok, i'll try to investigate this futher and maybe create a PR.
I think it's the same issue again that was in 7.0.0 also.
When we upgraded to 7.0.0 there was also compiler error because of kotlin version.
Here is the old compiler issue when 7.0.0 was released: https://github.com/flutter-stripe/flutter_stripe/issues/1022

**Comment by remonh87** [comment]
How did you solve it? 

It is really annoying (especially it outside our control) I will try to contact Stripe if they can help mitigating this in future

**Comment by remonh87** [comment]
hmm I can reproduce it as well now will look into it

**Comment by remonh87** [comment]
Fixed in `stripe_android 8.0.0+1`

**Comment by asaarnak** [comment]
@remonh87 Similar issue in 9.0.0:

e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-9.0.0/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt:808:14 Expression 'size' of type 'Int' cannot be invoked as a function. The function 'invoke()' is not found
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-9.0.0/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt:809:127 Expression 'size' of type 'Int' cannot be invoked as a function. The function 'invoke()' is not found
